openml/datasets/functions.py

```diff
@@ -425,7 +426,10 @@ def get_dataset(
 
         arff_file = _get_dataset_arff(description) if download_data else None
         if "oml:minio_url" in description and download_data:
-            parquet_file = _get_dataset_parquet(description)
+            try:
+                parquet_file = _get_dataset_parquet(description)
+            except urllib3.exceptions.MaxRetryError:
```

Comment:
I wasn't sure if I should be conservative here, or just catch all errors. Considering we haven't had any issues otherwise I went with the general best practice of only minimally catching exceptions.

Reply:
Looks reasonable to me.